solana-programs/claimable-tokens/program/src/utils/program.rs

```diff
@@ -24,6 +24,36 @@ pub struct AddressPair {
     pub derive: Derived,
 }
 
+/// Sender nonce account seed
+pub const NONCE_ACCOUNT_PREFIX: &str = "N_";
+
+/// Finds a program address, using first 32 bytes of `pubkey` + `seed` as seed, and
+/// `base` as base
+pub fn find_program_address_with_seed(
```

Comment:
discussed irl, let's consolidate this into the find_base_address

Reply:
followed up IRL - 2 separate functions w/better naming